### PR TITLE
Prepare release 0.5.1

### PR DESCRIPTION
**Pull request by nedimtokic**
Prepare release 0.5.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.1] - Unreleased
+## [0.5.1] - 2023-05-05
+### Added
+- Add global event listener
```
Comment (harlem88) [inline CHANGELOG.md:9]: Add a new line between different section
```suggestion
- Add global event listener

```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (nedimtokic) [reply CHANGELOG.md]: Added a new line and changed the message.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.5.1] - Unreleased
+## [0.5.1] - 2023-05-05
+### Added
+- Add global event listener for listening incoming data from Astarte.
```
Comment (harlem88) [inline CHANGELOG.md:9]:
```suggestion
- Add a global event listener for listening incoming data from Astarte.
```

**Review by harlem88 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (nedimtokic) [reply CHANGELOG.md]: Changed.

**Pull request merged**